_ml-commons-plugin/cluster-settings.md

```diff
@@ -364,22 +364,40 @@ When set to `true`, this setting enables the ability to run Python-based models
 
 ### Setting
 
-```
+```yaml
 plugins.ml_commons.enable_inhouse_python_model: false
 ```
 
 ### Values
 
-- Default value: false
+- Default value: `false`
+- Valid values: `false`, `true`
+
+## Safely delete models
```

Comment:
This heading doesn't work. It should probably be "Safely deleting a model", as in the previous file, but if we're trying to use a different heading, how about "Safely deleting models"? Note that the link text on line 58 in the previous file would need to be changed as well.

Reply:
This heading is not in gerund to mimic the other headings' form.

Reply:
Ah ok, fine as is, then.